src/app/blocks/web-money/web-money.component.ts

```diff
@@ -33,22 +33,115 @@ function removePaymentPointer() {
   }
 };
 
+const COIL_CLIENT_ID = '19d9c8e7-3a37-4fd5-803c-057300f4354b';
+function uuid4() {
```

Comment:
we have the UUID library already as a dependency, you could use that to generate this uuid if you want instead of including uuid generating code. Something like:
```js
import uuid from 'uuid';
console.log(uuid.v4());
```

Reply:
Done, thanks. Will merge now.